### ISSUE TITLE
setup.php is no longer compatible with recent OutputHandler interface changes

### ISSUE THREAD

**Issue by Koricz** [nette-intellij/intellij-nette-tester#42]
`Fatal error: Class TeamCityOutputHandler contains 2 abstract methods and must therefore be declared abstract or implement the remaining methods (Tester\Runner\OutputHandler::prepare, Tester\Runner\OutputHandler::finish) in PhpStorm2017.2\config\plugins\intellij-nette-tester\setup.php on line 108
`

https://github.com/nette/tester/commit/e0005633ce8511c89041b6b3325a952b71cc61d2#diff-1620ea1b0d8bce05b6b848b418ca7d19

**Setup script** option in Run/Debug Configuration window not working. I tried to set path to my own updated script but the default path from user directory is always used. 

PHPStorm 2017.2.1, Windows 10 Pro

**Comment by jiripudil** [comment]
True. Tester 2.0.0 is not yet supported. I'm looking forward to making it work again soon, but I can't give any promises about when I manage to do so. It's quite some amount of work to properly 
support both 1.7 and 2.0 releases, and I've just returned from a long off-the-grid vacation, so I need to catch up with work things first. Apologies.

If you have a working version of the script, you can help me by sending a pull request 😉

**Comment by hrach** [comment]
Personally, I suggest supporting only the 2.0 version. I see no added value in supporting the old one.

**Comment by jiripudil** [comment]
@hrach I've thought about that too. The only thing that's bothering me is that it would be difficult for people to stay on a 1.7-compatible version if they need it for some reason.

**Comment by Koricz** [comment]
Hi, quick fix is available here https://gist.github.com/Koricz/4f87caebb8a663b908e7aa5832250d64#file-setup-php. I just tried it in the console because as I wrote, I can't setup another --script path in preferences window and even if I rewrite the script in plugins directory, script is always overwritten by the older version. If it will work in PhpStorm as expected, I can clean up the whole code a little bit and create pull request...

**Comment by Koricz** [comment]
Btw, there could be added e.g. some select box for tester 1.7.1 / 2.x / custom  OutputHandler setup.php and extension will be compatible for both versions. Just my 2 cents ;-)

**Comment by jiripudil** [comment]
I was thinking about automatic version detection. I once had it in the plugin, so it should not be difficult to bring it back, and it would be much more convenient 🙂

**Comment by renekliment** [comment]
I just wanna say first that I just stumbled upon this plugin yesterday. Ugh, I can't believe I didn't know of this sooner! Great work @jiripudil !

I am also experiencing this `nette/tester:v2.0.0` incompatibility.

As a temporary workaround, I have put @Koricz's version of `setup.php` in the plugin `.jar` and it works flawlessly. I've expected some sort of checksum verification by phpStorm, but there doesn't seem to be any.

I have also tried removing the write permissions to the `setup.php` and the parent directory for my system user to prevent phpStorm rewriting it, but then it just fails as could be somewhat expected. So I'm sticking to the modified plugin `.jar` for now.

If this could be somehow addressed in upstream, that would be awesome.

Thanks for the work again!
